app/react/Library/helpers/requestState.js

```diff
@@ -55,7 +55,13 @@ export default function requestState(request, globalResources, calculateTableCol
   const documentsRequest = request.set(
     tocGenerationUtils.aggregations(docsQuery, globalResources.settings.collection.toJS())
   );
-  const markersRequest = request.set({ ...docsQuery, geolocation: true });
+  const templatesWithGeolocation = globalResources.templates.find(template =>
+    template.get('properties').find(property => property.get('type') === 'geolocation')
+  );
+  const markersRequest = request.set({
+    ...docsQuery,
+    geolocation: templatesWithGeolocation && true,
+  });
```

Comment:
I think the change should be to avoid calling 'markersRequest' instead of passing geolocation parameter with undefined, currently, when there are no geolocation properties, there are two calls with the same params.